Kickstarter-iOS/DataSources/CommentRepliesDataSource.swift

```diff
@@ -36,4 +62,152 @@ internal final class CommentRepliesDataSource: ValueCellDataSource {
       assertionFailure("Unrecognized combo: \(cell), \(value).")
     }
   }
+
+  /// Determines if a `Comment` object is present in the data source at a given `IndexPath`.
```

Comment:
Maybe should be 
`///get a comment at indexpath if present`.
Or something similar.

Reply:
Updated.